### DISCUSSION TITLE
Should the weather page support showing hourly forecasts for the current day?

### DISCUSSION THREAD

**Discussion by kchiem** [Blackymas/NSPanel_HA_Blueprint#426] [Polls]
This poll is per https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/399#issuecomment-1465303349

Some weather entities provide hourly forecast, but this integration doesn't support it yet.  Please vote if you'd like to see this.

**Comment by edwardtfn** [comment]
- #399 
- #426 
- #831 
- #1228 
- #1233
